src/nimlsp.nim

```diff
@@ -212,22 +199,23 @@ if not fileExists(nimpath / "config/nim.cfg"):
 proc main(){.async.} =
   while true:
     try:
-      debugEcho "Trying to read frame"
+      debugLog "Trying to read frame"
       let frame = await ins.readFrame
-      debugEcho "Got frame:\n" & frame
+      debugLog "Got frame:" 
```

Comment:
A bit strange that these are two separate log levels

Reply:
I use debug level output debug sentence , info level for raw payload , it's pretty to me.